server/models/postgis/project.py

```diff
@@ -56,6 +56,7 @@ class Project(db.Model):
     license_id = db.Column(db.Integer, db.ForeignKey('licenses.id', name='fk_licenses'))
     geometry = db.Column(Geometry('MULTIPOLYGON', srid=4326))
     centroid = db.Column(Geometry('POINT', srid=4326))
+    task_creation_mode = db.Column(db.Integer, default=TaskCreationMode.GRID.value, nullable=False)
```

Comment:
Note: this column already exists in the database ([added through the migrations](https://github.com/hotosm/tasking-manager/blob/56d1d13056bbea342132cf6e78b12f01c73d5dd3/migrations/versions/7d55a089b5bc_.py)), but it wasn't mapped into the DB models.